scripts/populate_sm_analysis.py

```diff
@@ -116,9 +141,11 @@ def _populate_qc_analysis_entries(multiqc_json_path: Path):
             analysis_type='joint-calling',
             analysis_status='completed',
             target=cohort,
-            meta=cohort.get_job_attrs() | dict(
+            meta=cohort.get_job_attrs()
+            | dict(
                 sequencing_type=cohort.sequencing_type.value,
             ),
+            project_name='seqr',
```

Comment:
Is this a remnant from testing?

Reply:
Intentionally left here, joint-calling results should go into the `seqr` project (we joint-call across datasets)

Reply:
ah, that makes sense :)